Graph/TestsForGraph/runTests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+argumentForCheckingFails=0
+existingOfMake=0
+
+cd ../
+if [[ -f "./minDist" ]]
+then
+    existingOfMake=1
+fi
+
+make > /dev/null
+
+./minDist ./TestsForGraph/test1.txt > testForCheck.txt
```

Comment:
Your script must work from anywhere.
for example assume your repository is located in /home/matos/projects/SPLabLessons
The following methods for running tests must work
1) cd /home/matos/projects/SPLabLessons/Graph/TestsForGraph && ./runTests.sh
2) cd /home && ./matos/projects/SPLabLessons/Graph/TestsForGraph/runTests.sh

Use relative paths based on current script path